python/lsst/daf/butler/script/queryCollections.py

```diff
@@ -22,22 +22,30 @@
 from __future__ import annotations
 
 import itertools
+from typing import Iterable, Union
 
 from astropy.table import Table
 from numpy import array
 
 from .._butler import Butler
+from ..registry import CollectionType
 
 
-def queryCollections(repo, glob, collection_type, chains):
+
+def queryCollections(
+    repo: str,
+    glob: Union[ellipsis, Iterable[str]],
```

Comment:
To get MyPy to handle `Ellipsis` properly I've always had to import the workaround [here](https://github.com/lsst/daf_butler/blob/2036ad53e56a84472a511b129f493549de86bdfd/python/lsst/daf/butler/registry/wildcards.py#L53-L75).  I suspect this means that this script isn't being type checked anyway; if you want to enable that, there's probably an ignore pattern in `mypy.ini` where an exception needs to be made.  In any case, I'd recommend just annotating this as `typing.Any`; the set of types accepted as collection expressions is actually large enough that I don't think it's worth spelling out for static checking.